doc/nrf/gs_modifying.rst

```diff
@@ -84,54 +93,115 @@ The following CMake commands can be managed by SES, if they target the ``app`` l
 
 The :file:`CMakeLists.txt` files for the sample applications in the :file:`sdk-nrf` repository already have the required tags.
 Therefore, if you always use SES to maintain them, you do not need to worry about tagging.
-Typically, the :file:`CMakeLists.txt` files include at least the :file:`main.c` file as source::
+Typically, the :file:`CMakeLists.txt` files include at least the :file:`main.c` file as source:
+
+.. code-block:: c
 
    # NORDIC SDK APP START
    target_sources(app PRIVATE src/main.c)
    # NORDIC SDK APP END
 
+Advanced compiler settings
+==========================
+
+The application has full control over the build process.
+
+Using Zephyr's configuration options is the standard way of controlling how the system is built.
+These options can be found under Zephyr's menuconfig :guilabel:`Build and Link Features` > :guilabel:`Compiler Options`.
+For example, to optimize for size, select :kconfig:`CONFIG_SIZE_OPTIMIZATIONS`; to compile for speed, select :kconfig:`CONFIG_SPEED_OPTIMIZATIONS`.
+
+If you want to specify non-standard compiler options, pass them using :kconfig:`CONFIG_COMPILER_OPT`.
```

Comment:
what is `non-standard compiler options` ?

If you really think an `Advanced compiler settings` belongs in a getting started guide, then I would like this re-phrased.

For example:
```suggestion
Compiler options not controlled by the Zephyr build system can be manually controlled through  :kconfig:`CONFIG_COMPILER_OPT`.
```